esp32h2-hal/examples/hello_world.rs

```diff
@@ -1,21 +1,55 @@
+//! This shows how to write text to uart0.
+//! You can see the output with `espflash` if you provide the `--monitor` option
+
 #![no_std]
 #![no_main]
 
+use core::fmt::Write;
+
 use esp32h2_hal::{
-    clock::{ClockControl, CpuClock},
-    entry,
+    clock::ClockControl,
     peripherals::Peripherals,
     prelude::*,
+    timer::TimerGroup,
+    // Rtc,
+    Uart,
 };
 use esp_backtrace as _;
-use esp_println::println;
+use nb::block;
 
 #[entry]
 fn main() -> ! {
     let peripherals = Peripherals::take();
-    let system = peripherals.PCR.split();
-    let clocks = ClockControl::configure(system.clock_control, CpuClock::Clock96MHz).freeze();
+    let mut system = peripherals.PCR.split();
+    let clocks = ClockControl::boot_defaults(system.clock_control).freeze();
+
+    // Disable the watchdog timers. For the ESP32-H2, this includes the Super WDT,
+    // and the TIMG WDTs.
+    // let mut rtc = Rtc::new(peripherals.LP_CLKRST);
+    let timer_group0 = TimerGroup::new(
+        peripherals.TIMG0,
+        &clocks,
+        &mut system.peripheral_clock_control,
+    );
+    let mut timer0 = timer_group0.timer0;
+    // let mut wdt0 = timer_group0.wdt;
+    let timer_group1 = TimerGroup::new(
+        peripherals.TIMG1,
+        &clocks,
+        &mut system.peripheral_clock_control,
+    );
+    // let mut wdt1 = timer_group1.wdt;
+
+    // rtc.swd.disable();
+    // rtc.rwdt.disable();
```

Comment:
Are these meant to me commented out?

Reply:
Yes, @playfulFence is still working on the `WDT` implementation